main/guides/zoe/contract-basics.md

```diff
@@ -1,46 +1,56 @@
 # Smart Contract Basics
 
 Before we look at how to make a contract such as the one in [the
-basic dapp](../getting-started/) in the previous section, let's cover some basics.
+basic dapp](../getting-started/) in the previous section, let's cover some basics by writing a simple contract that returns a greetings message. We will simply call it _greetings smart contract_. 
 
-A contract is defined by a JavaScript module that exports a `start` function
-that implements the contract's API.
+A contract is defined by a JavaScript module that exports a `start` function. For our greetings smart contract, the declaration of `start` function looks like this:
 
 <<< @/../snippets/zoe/src/01-hello.js#start
 
-Let's start with a contract with a simple `greet` function:
+Apart from `start`, we can have extra functions in our contract. These functions may be defined outside or within the `start` function depending on whether they need access to the variables defined in the `start` function.  For the greetings smart contract, we will have a simple `greet` function (outside `start`) that takes a string as a parameter (for example, name of the person calling the function) and returns a customized greeting message.
```

Comment:
> In addition to start, we can include additional functions in our contract. These functions can be defined either outside or within the start function, depending on whether they need access to variables defined in start. 

This can be reduced/concise for users who probably might already be familiar with basic JavaScript concepts.

Reply:
Let us talk about it. I am a bit inclined to err towards redundancy in the first tutorial.

Reply:
I made it short after our discussion, as below: 

For the greetings smart contract, we will have a simple `greet` function apart from `start` function. The `greet` function takes a string as a parameter (for example, name of the person calling the function) and returns a customized greeting message.